lib/src/client.dart

```diff
@@ -1652,8 +1652,8 @@ class Client extends MatrixApi {
 
   /// Immediately start a sync and wait for completion.
   /// If there is an active sync already, wait for the active sync instead.
-  Future<void> oneShotSync() {
-    return _sync();
+  Future<void> oneShotSync({Duration? timeout}) {
+    return _sync(timeout: timeout);
```

Comment:
This is unused now?

Reply:
yes but I don't see why not to keep it

Reply:
We shouldn't keep dead code, it just adds friction later imo. If there is no usecase for this, it shouldn't be there. Especially since this is a different timeout than people expect, imo.

Reply:
hm oki